ParquetCSVImporter/ClassMaps/CsvReaderExtensions.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+using CsvHelper;
+using ParquetClassLibrary.Utilities;
+using ParquetClassLibrary.Sandbox.Parquets;
+
+namespace ParquetCSVImporter.ClassMaps
+{
+    /// <summary>
+    /// Provides extensions to the CSV Reader.
+    /// </summary>
+    public static class CsvReaderExtensions
+    {
+        /// <summary>
+        /// Gets a set of records from the CSV Reader using the shim class that corresponds to the given type.
+        /// </summary>
+        /// <param name="in_csv">In CSV Reader.</param>
+        /// <typeparam name="T">The type of the record.</typeparam>
+        /// <returns>The records.</returns>
+        public static IEnumerable<T> GetRecordsViaShim<T>(this CsvReader in_csv) where T : ParquetParent
```

Comment:
isabellism: I wish I could find a way to say, "where T inherits from ParquetParent, but isn't." As is, you could pass in ParquetParent, which would fail at run time. I don't think that exists in C#, and my quick searches have failed to find it.